langfuse/task_manager.py

```diff
@@ -107,13 +113,37 @@ def _next(self):
             try:
                 item = queue.get(block=True, timeout=self._flush_interval - elapsed)
 
+                # convert pydantic models to dicts
+                if "body" in item and isinstance(item["body"], pydantic.BaseModel):
+                    item["body"] = item["body"].dict()
```

Comment:
exclude_none=True is missing here i think. I think we should only send diffs to the backend.